llvm/test/CodeGen/AArch64/aarch64-svcount-basic.ll

```diff
@@ -0,0 +1,51 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; RUN: llc < %s -mtriple=aarch64-linux-gnu -mattr=+sve | FileCheck %s
+
+;; This file tests basic use of `target("aarch64.svcount")` works with only +sve.
```

Comment:
I think basic testing for svcount type is already covered by llvm/test/CodeGen/AArch64/sme-aarch64-svcount.ll, so perhaps you can downgrade its -mattr option instead?